packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -361,6 +378,11 @@ export default {
         return 'state-trashed'
       }
       return ''
+    },
+    async openSidebarSharePanel(space) {
+      await this.closeSidebar()
```

Comment:
Why is it needed to close the sidebar?

Reply:
Don't know why I added it, it's not :D

Reply:
Ahh, I found the issue again, it is. It seems to be necessary when toggling through the share panel in the spaces overview (via the people-indicator). Otherwise, the share panel does not get updated.

Reply:
Isn't that an indicator for data not properly watched in the sharing panel? Can you look into that after merge? Closing and re-opening the sidebar feels a bit glitchy and might cause side effects in the future.